Libplanet/Crypto/IECPublicKey.cs

```diff
@@ -0,0 +1,36 @@
+using System.Collections.Generic;
+using Org.BouncyCastle.Crypto.Parameters;
+
+namespace Libplanet.Crypto
+{
+    /// <summary>
+    /// The interface of Elliptic Curve Public Key. It allows to align the essential methods and
+    /// uses in <see cref="CryptoConfig"/> and <see cref="ICryptoBackend{T}"/>, mainly
+    /// <see cref="Verify"/> and <see cref="KeyBytes"/>. This should be
+    /// inherited if there's a new implementation for Elliptic Curve Public Key.
+    /// </summary>
+    // ReSharper disable once InconsistentNaming
+    public interface IECPublicKey
```

Comment:
What's the reason for `IECPublicKey` instead of `IPublicKey`?
I'd say `EC` is too generic of a term for its meaning to be apparent. 🙄

Reply:
Amended at https://github.com/planetarium/libplanet/pull/2223!